Misc/NEWS.d/next/Documentation/2019-04-17-20-21-32.bpo-36651.3iUjvY.rst

```diff
@@ -0,0 +1 @@
+bpo-36651: Fixed Asyncio Event Loop documentation inconsistency
```

Comment:
There is an error in Travis due to this blurb entry at https://travis-ci.org/python/cpython/jobs/521431229#L309 . I am not sure this doc change needs a news entry.

```
blurb.BlurbError: Error in Misc/NEWS.d/next/Documentation/2019-04-17-20-21-32.bpo-36651.3iUjvY.rst:0:
Blurb 'body' can't start with 'bpo-'!
```